lib/widget/dropdown.dart

```diff
@@ -210,14 +238,25 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
     if (items != null) {
       results = [];
       for (SelectOneItem item in items) {
-        item.icon != null
-            ? results.add(
+        item.isIcon != null || item.isIcon == true
+            ?
+        results.add(
           DropdownMenuItem(
             child: Row(
               children: [
-                iconframework.Icon.fromModel(item.icon!),
-                const SizedBox(
-                  width: 10.0,
+                item.icon != null
+                    ? iconframework.Icon.fromModel(item.icon!)
+                    : SizedBox(
+                  width: widget._controller.size.toDouble(),
+                ),
+                SizedBox(
```

Comment:
This logic is long and hard to understand. Can you assign it to a variable called 'gap' and add the variable instead?

Reply:
okay

Reply:
I meant the 2nd Sizebox (starting line 252) is the gap.

Reply:
@vusters Basically I’m using first SizedBox with that icon because if icon exist in that list then give it specific gap and the second SizedBox is for if icon is available in list and it’s size is available or not available then it can give gap between text and icon